client/src/store/index.ts

```diff
@@ -686,6 +679,16 @@ const {
         },
       });
     },
+    async getFrame({ state, dispatch }, { frameId, projectId }) {
+      if (!frameId) {
+        return undefined;
+      } if (!state.frames[frameId]) {
```

Comment:
I'm surprised this passes linting
```suggestion
      }
      if (!state.frames[frameId]) {
```